technical-reports/resolver/rationale.md

```diff
@@ -1,48 +0,0 @@
-## Rationale
-
-### Context and Problem Statement
-
-In design systems, especially those at scale, managing multiple themes, modes, brands, and other modifiers can become complex. Designers and developers often struggle with:
```

Comment:
This is a continuation of other points, but we are using terms without defining them, including `themes`, `modes`, `brands`, and `modifiers`. These terms, until defined, hold no meaning, therefore the following statements don’t hold weight.

> Designers and developers often struggle with:

This is an opinion statement not backed by data. It goes without saying it _motivated_ the specification, however, there are 2 reasons I’d like to remove it:

1. **It’s irrelevant to the introduction**. Even assuming this is true, this does not help readers understand the following content any better, and
2. **It‘s irrelevant to the specification.** These claims are not normative, and are irrelevant to any syntax or concepts following.